orchestrator/careplanservice/handle_getcareplan.go

```diff
@@ -34,9 +34,11 @@ func (s *Service) handleGetCarePlan(ctx context.Context, id string, headers *fhi
 func (s *Service) handleSearchCarePlan(ctx context.Context, queryParams url.Values, headers *fhirclient.Headers) (*fhir.Bundle, error) {
 	params := []fhirclient.Option{}
 	for k, v := range queryParams {
-		// Skip param to include CareTeam since we need to add this for validation anyway
-		if k == "_include" && v[0] == "CarePlan:care-team" {
-			continue
+		for _, value := range v {
+			// Skip param to include CareTeam since we need to add this for validation anyway
+			if k == "_include" && value == "CarePlan:care-team" {
+				continue
+			}
 		}
 		params = append(params, fhirclient.QueryParam(k, v[0]))
```

Comment:
This should be removed and `params = append(params, fhirclient.QueryParam(k, value))` should be added into the range loop otherwise we might run into this problem again in the future with CarePlan search parameters